catroid/res/values-ar/strings.xml

```diff
@@ -391,11 +391,11 @@
   <string name="stage_dialog_maximize">زيادة الامتداد\n</string>
   <string name="stage_dialog_screenshot">لقطة للشاشة</string>
   <!-- Broadcast spinner -->
-  <string name="new_broadcast_message">جديد&#8230;</string>
+  <string name="new_broadcast_message">جديد…</string>
```

Comment:
Arabic is spelled from left to right, therefore you should've add the character on the left hand side. Furthermore adding the characters to the translations does not really help, as they are overwritten by Crowdin. The best way to go about changing this would be to change the character in the default strings.xml and then add it in Crowdin's system.